services/github/test/webhook_test.js

```diff
@@ -87,7 +89,156 @@ helper.secrets.mockSuite(testing.suiteName(), [], function(mock, skipping) {
   // Bad data: should all return 400 responses
   statusTest('Push without secret', 'webhook.push.no_secret.json', 400);
   statusTest('Unknown Event', 'webhook.unknown_event.json', 400);
-  statusTest('Push with bad secret', 'webhook.push.bad_secret.json', 403);
-  statusTest('Release with bad secret', 'webhook.release.bad_secret.json', 403);
+  statusTest('Push with bad secret', 'webhook.push.bad_secret.json', 400); // Schema validation happens before signature check
```

Comment:
I think we might be loosing our tests - we wanted to check signature validation here, and the fact that status code changed, means that schema is too strict (or our sample webhook data might be wrong). 
Can you please make sure that both payloads get us `403`?

Reply:
Makes sense, this is moved back to `403` - https://github.com/taskcluster/taskcluster/pull/7976/files#diff-71ca69908f181ea1f058d958cb1655c20a720474430e1723cba3d17358a36734R93